examples/dt/neutron/3cont_2comp-ipv6-all-geneve-dvr-stf-rally

```diff
@@ -0,0 +1,62 @@
+# Deployed Topology 3cont_2comp-ipv6-all-geneve-dvr-stf-rally
+
```

Comment:
Just generic observation, most of the things in these 4 DTs looks common, so seems it would have been better having a base/common DT and other extend it. That way would be quite visible what unnecessary in extended DTs and clubbed into base when found necessary.

Reply:
In fact, my suggestion is to look at https://github.com/openstack-k8s-operators/architecture/pull/45 and see how these DTs can be added there. The components which are part of the DTs proposed there don't care much on how networking is configured, as long as it works!

Reply:
Agree. 
Current defaults are as follows(in addition to tls, dvr):

tenant network = geneve
public network = flat
control plane ip version = ipv4
mtu 1500
no standalone networkers

default neutron config does not have other above-mentioned specifics (e.g. ovn-router-flavors  etc.)
https://github.com/openstack-k8s-operators/neutron-operator/blob/main/templates/neutronapi/config/01-neutron.conf

Reply:
Marking this as Resolved, because this is mapped to unified job uni-delta-ipv6